### PR TITLE
Feature/gaap headers

### PR DESCRIPTION
**Pull request by freesteph**
# Description
Update headers to match GaaP headers. See https://github.com/alphagov/govwifi-product-page/pull/212 for a  full explanation.

# Screenshots
## Before (public v3)
localhost_8080_super_admin_organisations

## After (GaaP v3)
localhost_8080_super_admin_organisations (1)

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -49,5 +49,5 @@ group :development, :test do
   gem 'byebug', '~> 11'
   gem 'listen', '~> 3'
   gem 'pry'
-  gem 'rack-mini-profiler', github: "MiniProfiler/rack-mini-profiler", require: false
+  gem 'rack-mini-profiler', require: false
```
Comment (phantas) [inline Gemfile]: Sounds like you can also remove `git` from https://github.com/alphagov/govwifi-admin/blob/65580fd94da019c22b2fb0b51960bd90f5b27629/Dockerfile#L39, based on https://github.com/alphagov/govwifi-admin/commit/3bb369952064434c6427fb9269e489ed545129bb

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -2,6 +2,27 @@ $govuk-image-url-function: "image-url";
 $govuk-font-url-function: "font-url";
 
 @import "../../../node_modules/govuk-frontend/govuk/all";
+
+
+.govuk-header__content,
+.govuk-header__logo {
+  @include govuk-media-query($from: desktop) {
+    width: 50%;
```
Comment (phantas) [inline app/assets/stylesheets/application.scss]: out of curiosity, where is this 50% coming from?

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -54,24 +54,25 @@
                 class="govuk-header__logotype-crown"
                 xmlns="http://www.w3.org/2000/svg"
                 viewbox="0 0 132 97"
-                height="31"
-                width="35">
+                height="30"
```
Comment (phantas) [inline app/views/layouts/application.html.erb]: just checking, isn't the height 32? https://github.com/alphagov/product-page-example/blob/master/source/stylesheets/modules/_govuk-logo.scss

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -81,7 +82,7 @@
             Menu
           </button>
           <nav>
-            <ul id="navigation" class="govuk-header__navigation " aria-label="Top Level Navigation">
+            <ul id="navigation" class="govuk-header__navigation govuk-header__navigation--end" aria-label="Top Level Navigation">
```
Comment (phantas) [inline app/views/layouts/application.html.erb]: Admin is not "active" here:
Screen Shot 2019-08-27 at 20 06 29

do you also need something like this? https://github.com/alphagov/govwifi-product-page/blob/555747c93081cd771869fdb0e77e5009ab0300fd/config.rb#L50-L53

Comment (phantas) [reply app/views/layouts/application.html.erb]: Admin actually seems to disappear and get replaced by Sign out once you log in. Not sure what the accessibility says on that (although it's not on scope to change it now)

File: app/assets/stylesheets/application.scss (hunk 2 above)
Comment (freesteph) [reply app/assets/stylesheets/application.scss]: this is to make the banner fit on one line, it's adapted from https://github.com/alphagov/product-page-example/blob/transition-to-govuk-frontend/source/stylesheets/modules/_header.scss

File: app/views/layouts/application.html.erb (hunk 3 above)
Comment (freesteph) [reply app/views/layouts/application.html.erb]: I aligned it with what we have in the product page repo, but happy to align onto whichever is the correct one. :triangular_ruler:

File: app/views/layouts/application.html.erb (hunk 4 above)
Comment (freesteph) [reply app/views/layouts/application.html.erb]: the active menu item functionality was done for the Product Page in the same PR as the new headers. In hindsight it was out of scope, so decided to not attempt it here. Are we happy with a new ticket to track that work?

File: Gemfile (hunk 1 above)
Comment (freesteph) [reply Gemfile]: done :heavy_check_mark:

File: app/views/layouts/application.html.erb (hunk 3 above)
Comment (phantas) [reply app/views/layouts/application.html.erb]: I defer it to you, you will know best what is the correct one. I am happy either way, if this is what's on the product page also.

**Pull request merged**